docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
+==================
+
+Hello hello, fellow documentarians! It's that time of year again: We’re very
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **May 6-8, 2018, in Portland, Oregon**.
+
+Every year, Write the Docs invites people from all across our community to come
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
```

Comment:
not just for writers? maybe rephrase to soemthing like "... or you're a support/engineering/role?? starting out / advocating for docs"? not sure but maybe we can discuss

Reply:
I'm happy to leave as is, all documentarians write docs innit? Just not all the time.

Reply:
I changed to 'working on', if that's cool.